apps/prairielearn/assets/scripts/lib/qrCodeButton.ts

```diff
@@ -1,29 +1,17 @@
 import QR from 'qrcode-svg';
-import { observe } from 'selector-observer';
 
 import { onDocumentReady, parseHTMLElement } from '@prairielearn/browser-utils';
 
-observe('.js-qrcode-button', {
```

Comment:
Why did you get rid of `observe`? What if there are multiple instances of `.js-qrcode-button` on the page? What if one is added dynamically?